container-engine-lib/lib/backend_impls/docker/docker_network_allocator/docker_network_allocator.go

```diff
@@ -28,88 +28,16 @@ const (
 	maxNumNetworkAllocationRetries = 10
 
 	timeBetweenNetworkCreationRetries = 1 * time.Second
+
+	allowedNetworkFirstOctet       = 10
+	maximumPossibleValueForAnOctet = 255
 )
 
 var networkCidrMask = net.CIDRMask(int(supportedIpAddrBitLength-networkWidthBits), int(supportedIpAddrBitLength))
 var networkWidthUint64 = uint64(math.Pow(float64(2), float64(networkWidthBits)))
 var maxUint32PlusOne = uint64(math.MaxUint32) + 1
 var emptyIpSet = map[string]bool{}
 
-// These IP ranges are reserved, so we'll skip creating any networks in them
-// If we don't, Docker will throw an error of "failed to set gateway while updating gateway: route for the gateway X.X.X.X could not be found: network is unreachable"
-// The start & end are BOTH inclusive (else we'd have no way to block off the single 255.255.255.255/32 address)
-// See https://en.wikipedia.org/wiki/IPv4#Special-use_addresses
-var disallowedIpRanges = [][][]byte{
-	{
-		// Current network (only valid as source address)
-		{0, 0, 0, 0}, {0, 255, 255, 255},
-	},
-	{
-		// Used for local communications within a private network
```

Comment:
We don't run into "failed to set gateway" even though we are now using this very network.

Wonder why this was disallowed